### PR TITLE
fix Jupyter metadata

### PR DESCRIPTION
**Pull request by jonsequitur**

### CODE REVIEW

File: src/Microsoft.DotNet.Interactive.Documents.Tests/JupyterFormatTests.cs [hunk 1]
```diff
@@ -81,19 +81,22 @@ public void notebook_metadata_default_language_is_honored_in_cells_without_langu
                 });
     }
 
-    [Fact]
-    public void Default_language_can_be_specified_in_metadata()
+    [Theory]
+    [InlineData("C#", "csharp")]
+    [InlineData("F#", "fsharp")]
+    [InlineData("f#", "fsharp")]
+    [InlineData("PowerShell", "powershell")]
```
Comment (brettfo) [inline src/Microsoft.DotNet.Interactive.Documents.Tests/JupyterFormatTests.cs:88]: isn't this kernel name `pwsh` and `powershell` an alias?

File: src/Microsoft.DotNet.Interactive.Documents.Tests/JupyterFormatTests.cs [hunk 2]
```diff
@@ -983,6 +986,11 @@ public void serialized_notebook_has_appropriate_metadata()
                     name = "C#",
                     pygments_lexer = "csharp",
                     version = "10.0"
+                },
+                dotnet_interactive = new
+                {
+                    defaultKernelName = "csharp",
+                    items = Array.Empty<object>()
```
Comment (brettfo) [inline src/Microsoft.DotNet.Interactive.Documents.Tests/JupyterFormatTests.cs:993]: if we default to `csharp`, then `items` should contain `{ name: "csharp", aliases: [] }`

File: src/Microsoft.DotNet.Interactive.Documents.Tests/JupyterFormatTests.cs (hunk 1 above)
Comment (jonsequitur) [reply src/Microsoft.DotNet.Interactive.Documents.Tests/JupyterFormatTests.cs:88]: Here's what we have in the kernelspec: 
image

File: src/Microsoft.DotNet.Interactive.Documents.Tests/JupyterFormatTests.cs (hunk 2 above)
Comment (jonsequitur) [reply src/Microsoft.DotNet.Interactive.Documents.Tests/JupyterFormatTests.cs:993]: I notice we don't have `language` in this object. Did you hit that in your PR?

**Review by colombod — APPROVED** [review]

**Pull request merged**